### ISSUE TITLE
Security issue in js-yaml dependency

### ISSUE THREAD

**Issue by amclin** [nickmerwin/node-coveralls#215]
NPM audit reports a security issue in the `js-yaml` dependency:
```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Denial of Service                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ js-yaml                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ coveralls [dev]                                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ coveralls > js-yaml                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/788                             │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

**Comment by julianlam** [comment]
https://www.npmjs.com/advisories/813

**Comment by XhmikosR** [comment]
This is fixed in 3.0.6 @nickmerwin. BTW you missed pushing the git tags to the repo.
